.github/workflows/all_plugins.yaml

```diff
@@ -1,5 +1,9 @@
 name: all_plugins
 
+concurrency:
+  group: all_plugins_${{ github.ref }}
```

Comment:
Do you know what would be the difference between ref, head_ref and ref_name? or why use `github.ref` instead of the other two.

Reply:
Just searched for [it](https://docs.github.com/en/actions/learn-github-actions/contexts#example-contents-of-the-github-context), ref is the whole path `/ref/heads/<branch-name>`. `head_ref` is available just on pr open... I think that `github.ref` is more general but open to using the other ones.